CHANGELOG.md

```diff
@@ -1,5 +1,108 @@
 # Unity Converters for Newtonsoft.Json changelog
 
-## x.0.1
+## 1.0.0
 
 - ✨ Initial release
+
+- ❤ Big love from these authors allowing me to take inspiration from their
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/manual/jilleJr/Newtonsoft.Json-for-Unity.Converters/pullRequest?prid=5461394)